src/main/java/com/tabnine/binary/fetch/BootstrapperSupport.java

```diff
@@ -14,9 +16,14 @@ static Optional<BinaryVersion> bootstrapVersion(
       BundleDownloader bundleDownloader) {
     Optional<BinaryVersion> localBootstrapVersion =
         locateLocalBootstrapSupportedVersion(localBinaryVersions);
-    return localBootstrapVersion.isPresent()
-        ? localBootstrapVersion
-        : downloadRemoteVersion(binaryRemoteSource, bundleDownloader);
+    if (localBootstrapVersion.isPresent()) {
+      return localBootstrapVersion;
+    }
+    if (ApplicationManager.getApplication() != null
+        && !ApplicationManager.getApplication().isUnitTestMode()) {
+      GettingStartedManagerKt.handleFirstTimePreview();
```

Comment:
no for now, but I suggest refactoring this, this dependency is really weird here, not sure I understand how version fetching is related to getting started...  maybe raise here some "just-installed" event here and subscribe to it somewhere